Assets/_BForBoss/Scripts/UserInterface/TabbedPanelViewBehaviour.cs

```diff
@@ -0,0 +1,67 @@
+using System;
+using PerigonGames;
+using UnityEngine;
+
+namespace BForBoss
+{
+    public sealed class TabbedPanelViewBehaviour : MonoBehaviour
```

Comment:
this is for Tabs UI 

So it's transferable to different games.  Click on a tab and open up a content

Reply:
should use PerigonTools NameSpace instead?

Reply:
I have a separate ticket for that